src/dbg_stack_trace/stack.cpp

```diff
@@ -293,7 +300,12 @@ namespace
 
                 if (func.empty())
                 {
-                    DWORD displacement = 0; // dummy variable
+                    #if defined(__amd64__) || defined(__x86_64__) // 64-bit
+                        DWORD64 displacement = 0; // dummy variable
+                    #elif defined(__i386__) || defined(_X86_) // 32-bit
+                        DWORD displacement = 0; // dummy variable
+                    #endif
```

Comment:
In the `#else` case right below (i can't mark it so i mark this block instead) there is that very same `displacement` variable, shouldn't its type changed depending on architecture like in this block?

Reply:
You are right, I think it should, I haven't notice it, because I compiled my forked repo using MinGW-w64 that is the `#if defined(__MINGW32__)`. If using `MSVC`, then the `#else` `DWORD displacement = 0; // dummy variable` should be patched too using `MSVC` specific pre-processor, because I don't think we are targeting `ARM` and not known compiler for the moment.

Reply:
Alongside with `MinGW-w64`, I have only `Visual Studio 2010` installed which have no `C++11` support to compile, I need a `Visual Studio 2015+` tester, or I am gonna apply the missing fixes anyway!

Reply:
There is currently no easy way to make a 64 bit build with MSVC, i'm trying to work on that, then i can build with MSVC 2019